scripts/dotnet-cli-build/PackageTargets.cs

```diff
@@ -97,6 +100,8 @@ public static BuildTargetResult CopySharedFxLayout(BuildTargetContext c)
 
             Directory.CreateDirectory(sharedFxRoot);
             Utils.CopyDirectoryRecursively(Path.Combine(Dirs.Stage2, "shared"), sharedFxRoot, true);
+            FixPermissions(sharedFxRoot);
```

Comment:
Should we fix these permissions before copying them to stage2? That way the stage2 directory has the correct perms too